cg/store/models.py

```diff
@@ -446,6 +447,10 @@ class Case(Base, PriorityMixin):
     priority: Mapped[Priority] = mapped_column(
         default=Priority.standard,
     )
+
+    sequencing_qc_status: Mapped[str | None] = mapped_column(
```

Comment:
```suggestion
    sequencing_qc_status: Mapped[SequencingQCStatus | None] = mapped_column(
```
I think this should be the mapped enum

Reply:
Resolved in https://github.com/Clinical-Genomics/cg/pull/3279/commits/4c83668d722edcb0335eec2a704e331e3b13e860.